### PR TITLE
Feature/boilerplate generation

### PR DESCRIPTION
**Pull request by CarlHembrough**
- API project boilerplate code generated
- Added empty lint scripts

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
We may need to change this to a more descriptive repo name i.e. bulletin-content or similar, can this be done later down the line ?

**Comment by CarlHembrough** [comment]
> We may need to change this to a more descriptive repo name i.e. bulletin-content or similar, can this be done later down the line ?

Yes - normally it is quite a bit of work to rename repo's, but as this is a prototype we are intentionally leaving out a lot of the repo setup until the design is settled.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Mac
+.DS_Store
+
+#VSCode
+.vscode
+
+# Intellij
+.idea/
+*.iml
+*.iws
```
Comment (redhug1) [inline .gitignore]: Add new line

File: ci/scripts/audit.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+export cwd=$(pwd)
+
+pushd $cwd/dp-content-api
+  make audit
+popd
```
Comment (redhug1) [inline ci/scripts/audit.sh]: add new line

File: dp-content-api.nomad [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        source      = "${NOMAD_TASK_DIR}/vars-template"
+        destination = "${NOMAD_TASK_DIR}/vars"
+      }
+
+      vault {
+        policies = ["dp-content-api-publishing"]
+      }
+    }
+  }
+}
```
Comment (redhug1) [inline dp-content-api.nomad]: add new line

File: features/helloworld.feature [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Feature: Helloworld
+
+
+
+  Scenario: Posting and checking a response
+    When I GET "/hello"
+    Then I should receive a hello-world response
```
Comment (redhug1) [inline features/helloworld.feature]: add new line

File: swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+        description: "The last health check date and time of the external service"
+        example: "2020-06-11T11:49:50.330089Z"
+      last_success:
+        type: string
+        description: "The last successful health check date and time of the external service"
+        example: "2020-06-11T11:49:50.330089Z"
+      last_failure:
+        type: string
+        description: "The last failed health check date and time of the external service"
+        example: "2019-09-22T11:48:51.0000001Z"
```
Comment (redhug1) [inline swagger.yaml]: add new line

File: Makefile [hunk 6]
```diff
@@ -32,4 +32,8 @@ convey:
 
 .PHONY: test-component
 test-component:
-	go test -cover -coverpkg=github.com/ONSdigital/dp-content-api/... -component
\ No newline at end of file
+	go test -cover -coverpkg=github.com/ONSdigital/dp-content-api/... -component
+
+.PHONY: lint
+lint:
+	exit
```
Comment (redhug1) [inline Makefile]: add new line

**Review by redhug1 — COMMENTED** [review]
see single comments

**Review by redhug1 — APPROVED** [review]
LGTM

**Pull request merged**